packages/@react-spectrum/label/test/StaticField.test.js

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {CalendarDate, CalendarDateTime, Time, ZonedDateTime} from '@internationalized/date';
+import React from 'react';
+import {render} from '@react-spectrum/test-utils';
+import {StaticField} from '../src';
+
+describe('StaticField', function () {
```

Comment:
It would be a good idea to include a test that checks that the user provided format options actually affect the rendered content